src/components/utils/popover/components/popover-item/popover-item-default/popover-item-default.ts

```diff
@@ -10,7 +13,7 @@ import { css } from './popover-item.const';
  * @todo replace multiple make() usages with constructing separate instaces
  * @todo split regular popover item and popover item with confirmation to separate classes
  */
-export class PopoverItem {
+export class PopoverItemDefault {
```

Comment:
how about adding a `PopoverItemBase` to extend?

Reply:
PopoverItemDefault and PopoverItemDelimiter have no common properties or logic. Why do this? To prevent [this](https://github.com/codex-team/editor.js/pull/2690/files#diff-bb2e77aad45e3f4d59ad2c01da87c477268ae156fa8bbfa9853f195f862ec7cdR10)?

Reply:
- because the both represents a popover item. The code will reflect the real-world rules.
- they can have common properties and methods if needed, like "hide", "show", etc